ftva_lab_data/views.py

```diff
@@ -90,6 +91,8 @@ def edit_item(request: HttpRequest, item_id: int) -> HttpResponse:
         "title": "Edit Item",
         "button_text": "Save Changes",
         "search": search,
+        # Encode search string here to make it safe for use in URL query strings
+        "url_encoded_search": urlencode({"search": search}),
```

Comment:
See my comment about simplifying the templates by moving parameter concatenation to the views, and passing it via the contexts.

Since there's repeated code here too, with something similar happening in many views, pull it into a simple method in `view_utils.py`.  I'd suggest something like `build_url_parameters(**kwargs)` or similar, which just unpacks `**kwargs`, applies special handling if needed... though it seems like the entire list of URL parameters could be encoded (rather, the concatenated string could be encoded).  

That's a simple way to make future changes easier.  If we need to add another parameter, which may or not require encoding, the logic is all in one place and may require no changes.